### ISSUE TITLE
chore: socketclient.ask should be returning the event listener, not just `true`.

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1739]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.2.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.1.1...4.2.0) (2021-06-06)

- build(deps): bump puppeteer from 9.1.1 to 10.0.0 [`#1742`](https://github.com/open-wa/wa-automate-nodejs/pull/1742)
- build(deps): bump swagger-ui-dist from 3.49.0 to 3.50.0 [`#1758`](https://github.com/open-wa/wa-automate-nodejs/pull/1758)
- Release 4.2.0 [`9c10032`](https://github.com/open-wa/wa-automate-nodejs/commit/9c1003217a5e6b431cbd5e8048e166294cbcfee8)
- ✨ SocketClient: implemented `stopListener` #1739 [`d280421`](https://github.com/open-wa/wa-automate-nodejs/commit/d280421b3348dc474c99b25237b474174d34f2fc)
- ✨ SocketClient: implemented remote access to `ev` #1760 [`51d940a`](https://github.com/open-wa/wa-automate-nodejs/commit/51d940ac30a3fb1483bba18dc6e80b8af01cc08d)
